provider/gce/config.go

```diff
@@ -11,17 +11,16 @@ import (
 	"github.com/juju/juju/environs/config"
 )
 
-// TODO(ericsnow) While not strictly config-related, we could use some
-// mechanism by which we can validate the values we've hard-coded in
-// this provider match up with the external authoritative sources. One
-// example of this is the data stored in instancetypes.go. Similarly
-// we should also ensure the cloud-images metadata is correct and
-// up-to-date, though that is more the responsibility of that team.
-// Regardless, it may be useful to include a tool somewhere in juju
-// that we can use to validate this provider's potentially out-of-date
-// data.
+const (
+	RelativeImagePath = "relative-image-path"
+)
 
-var configSchema = environschema.Fields{}
+var configSchema = environschema.Fields{
+	RelativeImagePath: {
+		Description: "Base path to look for machine disk images. Setting this to a custom value in conjunction with setting the default-image-id allows you to provide custom virtual machine images.",
```

Comment:
"default-image-id" is not an attribute that can be set for GCE providers, so I don't quite follow this text.

Perhaps all that's needed is "Base path to look for machine disk images"